frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -23,4 +23,8 @@ export type OverviewItem<T = K8sResourceKind> = {
   revisions?: K8sResourceKind[];
 };
 
+export type OperatorBackedServiceKindMap = {
```

Comment:
Why is this in shared?
Seems very specific to topology.

Reply:
Its being used by `knative-plugin` as well that's why I had to add it to `console-shared`.

Reply:
But knative-plugin depends on dev-console for contributing to topology.
You should be enhancing the `TopologyDataObject`.

Reply:
It is not actually part of `TopologyDataObject`. So, moved the type definition to `topology-types.ts` file.

Reply:
You're right. I misread where it was. But `topology-types.ts` is a better home for it.